index.html

```diff
@@ -64,9 +64,12 @@
           <div id="facets" class="tab-pane control control-survey active"></div>
           <div id="listTab" class="tab-pane control control-sidebar">
             <div class="list-filters">
-              <select id="select_county">
+              <select id="select_county" class="form-control">
                 <option value="" selected>All Counties</option>
               </select>
+              <div class="btn btn-default btn-print" title="Print" onClick="window.print();">
+                🖶 Print
```

Comment:
Hmmm. I can see it. It's the utf8 printer icon .... maybe this is safer? &#128438;